css-media-queries/breakpoints.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Media Queries - Breakpoints</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+      }
+
+      .card-wrapper {
+        padding: 0.5rem;
+        flex-basis: 100%;
+      }
+
+      /* your code here 👇 */
+      .card-wrapper {
+        @media (min-width: 576px) {
+          flex-basis: 50%;
+        }
+      }
+
+      .card-wrapper {
+        @media (min-width: 768px) {
+          flex-basis: calc(100% / 3);
+        }
+      }
```

Comment:
These should be flipped. `@media` should be first, and the `.card-wrapper` selector should be inside the media query